docs/spec/api.md

```diff
@@ -618,26 +618,6 @@ The "digest" parameter must be included with the PUT request. Please see the
 [_Completed Upload_](#completed-upload) section for details on the parameters
 and expected responses.
 
-Additionally, the upload can be completed with a single `POST` request to
```

Comment:
@SvenDowideit You'll have to ask @RichardScothern for the current release process.

I'd like to find something that allows us to release multiple versions of the docs and preserves the lineage between branches (something that cherry picks break).